sdks/java/extensions/protobuf/src/test/java/org/apache/beam/sdk/extensions/protobuf/ProtoCoderTest.java

```diff
@@ -167,4 +170,26 @@ public void testNonDeterministicProperty() throws CoderException {
     Coder<MessageWithMap> coder = ProtoCoder.of(MessageWithMap.class);
     assertNotEquals(CoderUtils.encodeToBase64(coder, msg2), CoderUtils.encodeToBase64(coder, msg1));
   }
+
+  @Test
+  public void testDynamicMessage() throws Exception {
```

Comment:
Would it be too onerous to add a few more tests here? Maybe set some more fields and some nested fields?

Reply:
Added another test `testDynamicNestedRepeatedMessage` that adds another repeated nested message.